config/config.go

```diff
@@ -32,13 +34,14 @@ func MakeConfig() *types.OracleServiceConfig {
 	var pluginConfFile string
 	var gasTipCap uint64
 
-	flag.Uint64Var(&gasTipCap, "oracle_gas_tip_cap", DefaultGasTipCap, "The gas priority fee cap to issue the oracle data report transactions")
-	flag.StringVar(&pluginDir, "oracle_plugin_dir", DefaultPluginDir, "The DIR where the adapter plugins are stored")
-	flag.StringVar(&symbols, "oracle_symbols", DefaultSymbols, "The symbols string separated by comma")
-	flag.StringVar(&keyFile, "oracle_key_file", DefaultKeyFile, "Oracle server key file")
-	flag.StringVar(&autonityWSUrl, "oracle_autonity_ws_url", DefaultAutonityWSUrl, "WS-RPC server listening interface and port of the connected Autonity Go Client node")
-	flag.StringVar(&keyPassword, "oracle_key_password", DefaultKeyPassword, "Password to the oracle server key file")
-	flag.StringVar(&pluginConfFile, "oracle_plugin_conf", DefaultPluginConfFile, "The plugins' configuration file in YAML")
+	flag.IntVar(&logLevel, "log.level", DefaultLogVerbosity, "Set the logging level, available levels are:  0: NoLevel, 1: Trace, 2:Debug, 3: Info, 4: Warn, 5: Error")
+	flag.Uint64Var(&gasTipCap, "gas.tip.cap", DefaultGasTipCap, "Set the gas priority fee cap to issue the oracle data report transactions.")
+	flag.StringVar(&pluginDir, "plugin.dir", DefaultPluginDir, "Set the directory of the data plugins.")
+	flag.StringVar(&symbols, "symbols", DefaultSymbols, "Set the symbols string separated by comma")
+	flag.StringVar(&keyFile, "key.file", DefaultKeyFile, "Set oracle server key file")
+	flag.StringVar(&keyPassword, "key.password", DefaultKeyPassword, "Set the password to decrypt oracle server key file")
+	flag.StringVar(&autonityWSUrl, "autonity.ws.url", DefaultAutonityWSUrl, "Set the WS-RPC server listening interface and port of the connected Autonity Client node")
+	flag.StringVar(&pluginConfFile, "plugin.conf", DefaultPluginConfFile, "Set the plugins' configuration file")
 
 	flag.Parse()
 	if len(flag.Args()) == 1 && flag.Args()[0] == "version" {
```

Comment:
Ok I see we have already a version command, however it is not documented as an option. Therefore when using the oracle for testing r4 I did not find a way to determine which version of the oracle I was using. I think it should be documented as an option.

Reply:
Doc it at README in 997d7f49c97fa012e19b509165f138c3b00391aa